starport/services/networkbuilder/simulation.go

```diff
@@ -0,0 +1,219 @@
+package networkbuilder
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"context"
+	"github.com/cenkalti/backoff"
+	"github.com/otiai10/copy"
+	"github.com/pelletier/go-toml"
+	"github.com/tendermint/starport/starport/pkg/availableport"
+	"github.com/tendermint/starport/starport/pkg/cmdrunner"
+	"github.com/tendermint/starport/starport/pkg/cmdrunner/step"
+	"github.com/tendermint/starport/starport/pkg/cosmosver"
+	"github.com/tendermint/starport/starport/pkg/gomodulepath"
+	"github.com/tendermint/starport/starport/pkg/httpstatuschecker"
+	"github.com/tendermint/starport/starport/pkg/xurl"
+	"github.com/tendermint/starport/starport/services/chain"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const ErrValidatorSetNil = "validator set is nil in genesis and still empty after InitChain"
```

Comment:
This one kind of against Go conventions. Names that starts with an `Err` prefix should have an `error` type. Let's try to pick a different name?